### ISSUE TITLE
Viber compatibility issues

### ISSUE THREAD

**Issue by ameshkov** [AdguardTeam/AdguardForAndroid#170]
http://4pda.ru/forum/index.php?s=&showtopic=601691&view=findpost&p=44389157

@gshumihin please check it.

**Comment by gshumihin** [comment]
Tried it for couple hours. Forks fine for me.

**Comment by gshumihin** [comment]
Waiting for reply from user.

**Comment by Bohdan-SUP** [comment]
We have a log here:
IPL-209-96411

**Comment by ameshkov** [comment]
Suspicious records in the log:
```
12:06:22.493 [pool-3-thread-1312] WARN  c.a.f.proxy.BaseHttpsProxyServer - TCP id=3820 Cannot connect to /52.0.254.122:443
13:05:18.846 [pool-3-thread-1341] WARN  c.a.f.proxy.BaseHttpsProxyServer - TCP id=3936 Cannot connect to /52.0.252.115:443
```

I believe that this is the same issue as #43. Connection was redirected to a local proxy (and was accepted by that proxy), Viber considers that IP address is reachable and use it, while in fact IP address is unavailable.

This issue should be resolved along with #217.
